### PR TITLE
Crisprcasfinder module added

### PR DESCRIPTION
**Pull request by vikasguptaebi**

### CODE REVIEW

File: modules/ebi-metagenomics/crisprcasfinder/meta.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "crisprcasfinder"
+description: crisprcasfinder
+keywords:
+  - sort
```
Comment (vagkaratzas) [inline modules/ebi-metagenomics/crisprcasfinder/meta.yml:6]: replace with another keyword that actually does something here

File: modules/ebi-metagenomics/crisprcasfinder/meta.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+  - gff:
+      type: file
+      description: annotations file
+      pattern: "*.gff"
+  - tsv:
+      type: file
```
Comment (vagkaratzas) [inline modules/ebi-metagenomics/crisprcasfinder/meta.yml:40]: this is a folder, change approprietly

File: modules/ebi-metagenomics/crisprcasfinder/main.nf [hunk 3]
```diff
@@ -0,0 +1,55 @@
+process CRISPRCASFINDER {
+    tag "$meta.id"
+    label 'process_single'
+
+
```
Comment (vagkaratzas) [inline modules/ebi-metagenomics/crisprcasfinder/main.nf:5]: remove extra empty line here

File: modules/ebi-metagenomics/crisprcasfinder/main.nf [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    CRISPRCasFinder.pl -i $fasta \
+        -so /opt/CRISPRCasFinder/sel392v2.so \
+        -def G \
+        -drpt /opt/CRISPRCasFinder/supplementary_files/repeatDirection.tsv \
```
Comment (vagkaratzas) [inline modules/ebi-metagenomics/crisprcasfinder/main.nf:31]: Strip this down to the bare minimum

File: modules/ebi-metagenomics/crisprcasfinder/meta.yml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "crisprcasfinder"
+description: crisprcasfinder
+keywords:
+  - sort
+  - crisprcasfinder
+  - genomics
+tools:
+  - "crisprcasfinder":
+      description: "commandline tool"
```
Comment (vagkaratzas) [inline modules/ebi-metagenomics/crisprcasfinder/meta.yml:11]: Write a bit more on what it does

**Review by vagkaratzas — CHANGES_REQUESTED** [review]
Almost there! Very good first attempt

File: modules/ebi-metagenomics/crisprcasfinder/meta.yml (hunk 1 above)
Comment (vikasguptaebi) [reply modules/ebi-metagenomics/crisprcasfinder/meta.yml]: changed

File: modules/ebi-metagenomics/crisprcasfinder/meta.yml (hunk 2 above)
Comment (vikasguptaebi) [reply modules/ebi-metagenomics/crisprcasfinder/meta.yml]: changed

File: modules/ebi-metagenomics/crisprcasfinder/main.nf (hunk 3 above)
Comment (vikasguptaebi) [reply modules/ebi-metagenomics/crisprcasfinder/main.nf]: done

File: modules/ebi-metagenomics/crisprcasfinder/main.nf (hunk 4 above)
Comment (vikasguptaebi) [reply modules/ebi-metagenomics/crisprcasfinder/main.nf]: changed and added as config

File: modules/ebi-metagenomics/crisprcasfinder/meta.yml (hunk 5 above)
Comment (vikasguptaebi) [reply modules/ebi-metagenomics/crisprcasfinder/meta.yml]: changed

**Review by vagkaratzas — APPROVED** [review]
Nice :D

**Pull request merged**